control-plane/tests/test_phase21_runtime_auth_validation.py

```
@@ -230,6 +293,91 @@ def __init__(self, server_address: tuple[str, int], handler_class: type) -> None
                     servers[0].server_close()
                 thread.join(timeout=2)
 
+    def test_runtime_validation_rejects_placeholder_bound_runtime_credentials(self) -> None:
+        store, _ = make_store()
+        placeholder_secret_service = AegisOpsControlPlaneService(
+            RuntimeConfig(
+                host=TEST_NON_LOOPBACK_HOST,
+                port=8080,
+                postgres_dsn="postgresql://control-plane.local/aegisops",
+                wazuh_ingest_shared_secret="<set-me>",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "wazuh_ingest_shared_secret"

(S106)